### ISSUE TITLE
sosa:sample vs sosa:sampling issue

### ISSUE THREAD

**Issue by charlesvardeman** [ESIPFed/sweet#107]
I'm not sure if this is a typo or by design. The sweet-ssn mapping defines Sample as a subclass of sosa:Sampling:

```
# https://github.com/ESIPFed/sweet/blob/master/alignments/sweet-ssn-mapping.ttl
res:Sample
  rdfs:subClassOf sosa:Sampling ;
.
```

The sosa vocabulary defines sampling as a procedure, not the physical sample
```
# https://www.w3.org/TR/vocab-ssn/#SOSASampling
Sampling - An act of Sampling carries out a (Sampling) Procedure to create or transform one or more Samples.
```

However one of the only uses of res:Sample in Sweet is for IceCore
```
###  http://sweetontology.net/realmCryo/IceCore
cryo:IceCore rdf:type owl:Class ;
             rdfs:subClassOf res:Sample ,
                             [ rdf:type owl:Restriction ;
                               owl:onProperty srela:inside ;
                               owl:someValuesFrom water:Ice
                             ] .
```

Which seems to refer to the physical sample, not the procedure?

It may be better to align res:Sample to sosa:Sample?
```
# https://www.w3.org/TR/vocab-ssn/#SOSASample
Sample - Feature which is intended to be representative of a FeatureOfInterest on which Observations may be made. 
```

**Comment by lewismc** [comment]
I think this is a fair analysis... @charlesvardeman can you submit a PR? Thanks for chiming in, I wondered if we would get this sorted after ESIP so thank you for the following up.

**Comment by rduerr** [comment]
Well a sample is not sampling and an ice core is definitely a thing not a process - the SWEET definition of an IceCore as a Sample is appropriate - but if the SOSA definition is about the process, a different link is needed.

Happy to write this up as I haven’t seen an issue yet….

Ruth

**Comment by charlesvardeman** [comment]
Ok. I will make a pull request with the change as soon as I validate the ttl file. I also added some definitions in the form of rdfs:comment and skos:definition annotations so that we can populate LODE or other human-readable documentation. I would like to go back and do the same thing with relmCryo based on the definitions that Ruth had added to ENVO.

**Comment by rduerr** [comment]
That would be gratefully appreciated!  I would be happy to review those changes once made….

Ruth

**Comment by dr-shorthair** [comment]
The subsumption hierarchy is

res:Sample ⊆ res:ExperimentActivity ⊆ res:Experiment ⊆ res:Investigation ⊆ res:Research ⊆ huma:HumanActivity

which appears to be saying that res:Sample is using 'sample' as a verb rather than noun. 
So if it is used to refer to the artefact rather than the activity, then the subclass relationships are incorrect.

**Comment by cmungall** [comment]
disjointness axioms in both ontologies can be a great way of making these implicit assumptions explicit

**Comment by charlesvardeman** [comment]
@dr-shorthair That makes sense as to the original alignment. I may have missed it, but I don't see any class that refers to the actual artifact other than maybe sciprov:Artifact? Does this seem to imply that "res:Sample" should really be "res:Sampling", res:Sampling owl:equivalentClass stat:Sampling, and that there should be a res:Sample that is a subclass of sciprov:Artifact?

**Comment by lewismc** [comment]
@charlesvardeman right issue can corrected by https://github.com/ESIPFed/sweet/pull/109 correct?

**Comment by dr-shorthair** [comment]
@charlesvardeman is right. The original alignment of `res:Sample` to `sosa:Sampling` is consistent with the subsumption, but it is clear from its mis-use as the parent of ice-core that it would be better renamed to `res:Sampling`, and an additional class for Sample introduced.

**Comment by lewismc** [comment]
pull requests welome... ;)

**Comment by dr-shorthair** [comment]
I've dug a bit deeper. 

`res:Sample` is defined in humanResearch.ttl and from the subsumption relations `res:Sample ⊆ res:ExperimentActivity ⊆ res:Experiment ⊆ res:Investigation ⊆ res:Research ⊆ huma:HumanActivity` it clearly relates to 'the act of sampling'. 

It is cross-referenced in two places
- in realmCryo.ttl -              `cryo:IceCore  rdfs:subClassOf res:Sample .`
- in reprMathStatistics.ttl - `stat:StatisticalSample rdfs:subClassOf res:Sample .`

Meanwhile, also in reprMathStatistics.ttl there is `stat:Sampling rdfs:subClassOf stat:StatisticalOperation .` 

So there is clearly some confusion here. 

We could rename `res:Sample` to `res:Sampling` for conistency with both SOSA and `stat:Sampling`. But this might break existing usages that use it in the 'act of sampling in human research' sense. OTOH leaving it as-is also risks continuing the confusion.  

The notion of 'sample' in the sense of 'an artefact that is designed to be representative of something bigger' is actually more general than either physical or statistical sampling, as reflected by the attempt to find a common superclass of IceCore and StatisticalSample - its just that the one they chose (from human research) was not it! 

So on balance I think I would propose creating new classes for 'sample' and 'sampling`. This could be 
- `mod:Sample` and `mode:Sampling` in reprSciModel.ttl 
or 
- `meth:Sample` and `meth:Sampling` in reprSciMethdology.ttl
or maybe somewhere else. 
And then we can knot together the sub-class relationships mentioned above consistently. 
Note that `res:Sample rdfs:subClassOf mod:Sampling .` but no classes have to be renamed. 

Comments?

**Comment by dr-shorthair** [comment]
Also 
`mod:Sampling rdfs:subClassOf proc:Process .`
but I can't find a SWEET class for endurant/continuant that would be the (ultimate) super-class of `mod:Sample` ...

**Comment by dr-shorthair** [comment]
Proposed fix in https://github.com/ESIPFed/sweet/tree/ISSUE-107

**Comment by dr-shorthair** [comment]
Appears to be fixed by #127
